.github/workflows/coding-standards.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       matrix:
         php-version:
-          - "7.4"
+          - "8.0"
```

Comment:
Perhaps we should run it with 7.1, that's the compatibility we need to maintain.

Reply:
I've added 7.1 as well.

Reply:
:thinking: I think one run should be enough, if we add `<config name="php_version" value="70100"/>` to the configuration.

Reply:
Let's check that. 🙂

Reply:
Yes, that does the trick. I could even remove two more rules from the exclude list now. 🎉